### PR TITLE
fix grammar in docs

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: Doc/bytes_mode.rst [hunk 1]
```diff
@@ -55,7 +55,7 @@ argument to :func:`ldap.initialize`:
     Text values are represented as ``unicode``.
 
 If not given explicitly, python-ldap will default to ``bytes_mode=True``,
-but if an ``unicode`` value supplied to it, it will warn and use that value.
+but if an ``unicode`` value is supplied to it, it will warn and use that value.
```
Comment (graingert) [inline Doc/bytes_mode.rst]:
```suggestion
but if a ``unicode`` value is supplied to it, it will warn and use that value.
```

**Comment by graingert** [comment]
https://www.englishclub.com/pronunciation/a-an.htm

"ay you-ni-code"

**Comment by encukou** [comment]
Thanks for your attention to detail!

**Comment by graingert** [comment]
@encukou I've just read that line a few dozen times because I can't remember it :p

**Pull request merged**